Sources/Plasma/FeatureLib/pfPatcher/plManifests.cpp

```diff
@@ -54,22 +54,38 @@ Mead, WA   99021
 
 plFileName plManifest::ClientExecutable()
 {
+#if HS_BUILD_FOR_MACOS
+    return MANIFEST("plClient.app", "UruExplorer.app");
+#else
     return MANIFEST("plClient.exe", "UruExplorer.exe");
+#endif
 }
 
 plFileName plManifest::PatcherExecutable()
 {
+#if HS_BUILD_FOR_MACOS
+    return MANIFEST("plClient.app", "UruExplorer.app");
+#else
     return MANIFEST("plUruLauncher.exe", "UruLauncher.exe");
+#endif
 }
 
 ST::string plManifest::ClientManifest()
 {
+#if HS_BUILD_FOR_MACOS
+    return MANIFEST("MacThinInternal", "MacThinExternal");
+#else
     return MANIFEST("ThinInternal", "ThinExternal");
+#endif
 }
 
 ST::string plManifest::ClientImageManifest()
 {
+#if HS_BUILD_FOR_MACOS
+    return MANIFEST("MacInternal", "MacExternal");
```

Comment:
Minor thing: the old Mac manifests were called `macInternal` and `macExternal`. Should we match the old capitalization? (Not that it really matters, because the old Cider wrapper hasn't worked for years now, so nothing needs these specific names anymore...)

Reply:
Agreed